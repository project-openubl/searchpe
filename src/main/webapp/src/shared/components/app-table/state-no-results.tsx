import React from "react";
import {
  EmptyState,
  EmptyStateBody,
  EmptyStateIcon,
  EmptyStateVariant,
  Title,
} from "@patternfly/react-core";
import { SearchIcon } from "@patternfly/react-icons";

export const StateNoResults: React.FC = () => {
  return (
    <EmptyState variant={EmptyStateVariant.small}>
      <EmptyStateIcon icon={SearchIcon} />
      <Title headingLevel="h2" size="lg">
        No results found
      </Title>
      <EmptyStateBody>
        No results match the filter criteria. Remove all filters or clear all
        filters to show results.
      </EmptyStateBody>
    </EmptyState>
  );
};
