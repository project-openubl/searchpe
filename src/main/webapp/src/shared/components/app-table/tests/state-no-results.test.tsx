import React from "react";
import { shallow } from "enzyme";

import { StateNoResults } from "../state-no-results";

describe("StateNoResults", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<StateNoResults />);
    expect(wrapper).toMatchSnapshot();
  });
});
