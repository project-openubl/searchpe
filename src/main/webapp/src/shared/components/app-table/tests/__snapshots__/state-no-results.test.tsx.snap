// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateNoResults Renders without crashing 1`] = `
<EmptyState
  variant="small"
>
  <EmptyStateIcon
    icon={[Function]}
  />
  <Title
    headingLevel="h2"
    size="lg"
  >
    No results found
  </Title>
  <EmptyStateBody>
    No results match the filter criteria. Remove all filters or clear all filters to show results.
  </EmptyStateBody>
</EmptyState>
`;
